tasks/main.yml

```diff
@@ -1 +1,31 @@
 # role tasks
+
+- name: Install necessary packages
+  package:
+    name: parted
+    state: present
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+
+- name: Clear the disk signatures
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: "wipefs --all --force {{ item }}"
+  register: signatures_before
+  changed_when: '"offset" in signatures_before.stdout'
+  with_items: "{{ disk_wipe_disks }}"
+
+- name: Re-read partion tables
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: "partprobe {{ item }}"
+  changed_when: false
```

Comment:
Technically by changing the list of partitions that Linux can see, we are making a change here.

Reply:
updated